### ISSUE TITLE
10ch_MOD_i6_Programmer executable does not run

### ISSUE THREAD

**Issue by codeMonkeyDo** [benb0jangles/FlySky-i6-Mod-#28]
Hi, 

I'm running on Windows 10 Pro, 64-bit. 

The **10ch_MOD_i6_Programmer** executable does not run from any of the version directories. Attempting to run the executable causes a MSCOMM32.OSX file to be generated in the local directory (on disk size: 104KB). But that is it. No instance of the process can be seen in Taskmanager.

Attempting to run in compatibility mode sees no success, neither does running from the command prompt. 

If I had access to the source code I may be able to provide more insight. Where is the source code for this project?

**Comment by NicHyslop** [comment]
Try running the executable by right-clicking it and then selecting "Run as Administrator". That fixed the issue for me.

**Comment by codeMonkeyDo** [comment]
That was the problem - thank you. 

I checked the readme and I couldn't see this mentioned. Why does it need to be run with Admin Privilages? 

It's other wise ran and installed with no problems. 

I did notice that the modified firmware is based of a 2014 build, my Tx had a more current 2015 version on it. But that's probably another issue. As it stands it seems to have worked! Thank you to the dev's!
